Configurations/unix-Makefile.tmpl

```diff
@@ -686,8 +686,10 @@ tar:
 	DISTDIR=$(NAME); \
 	mkdir -p $$TMPDIR/$$DISTDIR; \
 	(cd $(SRCDIR); \
+	 submodule_re=`git submodule status | sed -e 's/^.//' | cut -d' ' -f2`; \
+	 submodule_re="^(`echo $$submodule_re | sed -e 's/ /|/g'`)\$$"; \
 	 git ls-tree -r --name-only --full-tree HEAD \
-         | grep -v '^fuzz/corpora' \
+	 | grep -v '^fuzz/corpora' | egrep -v "$$submodule_re" \
```

Comment:
Oh! Actually manual pages call egrep obsolescent and deprecated. But they say that if you have expression, then you should denote it with -E. By this logic egrep should go, but you should use -E. Leaning something new every day...